kmir/k-src/mir.md

```diff
@@ -61,16 +60,21 @@ The `#halt` construct is used to signify the end of execution. Any remaining ite
 endmodule
 ```
 
-
 `MIR-SYMBOLIC` is a stub module to be used with the Haskell backend in the future. It does not import `MIR-AMBIGUITIES`, since `amb` productions are not supported by the Haskell backend. We may need to consult the C semantics team when we start working on symbolic execution.
 
 ```k
 module MIR-SYMBOLIC
-  imports MIR-CONFIGURATION
-  imports MIR-INITIALIZATION
-  imports MIR-EXECUTION
-  imports MIR-FINALIZATION
+  // imports MIR-CONFIGURATION
+  // imports MIR-INITIALIZATION
+  // imports MIR-EXECUTION
+  // imports MIR-FINALIZATION
+  imports MIR
+
+    syntax KItem ::= runLemma ( Step ) | doneLemma ( Step )
```

Comment:
I believe that the RPC-based prover supports functional claims directly these days, so this run / done workaround is only strictly needed for the text-based `kprove` workflow. Feel free to leave it in for your own development purposes, but just making you aware!